hoomd/md/methods/__init__.py

```diff
@@ -13,5 +13,5 @@
 """
 
 from . import rattle
-from .methods import (Method, NVT, NPT, NPH, NVE, Langevin, Brownian, Berendsen,
+from .methods import (Method, Langevin, Brownian, ConstantVolume, ConstantPressure, ConstantEnergy, MTTKThermostat, BussiThermostat,
```

Comment:
Move the thermostats to a new file: `thermostat.py`.

Then we can name the classes without the `Thermostat` suffix:
* `Thermostat`
* `ConstantEnergy`
* `Berendsen`
* `Bussi`
* `MTTK`